### DISCUSSION TITLE
LGTMよりストック数を重視して欲しい

### DISCUSSION THREAD

**Discussion by lamrongol** [increments/qiita-discussions#241] [要望・アイデア・不具合]
現在LGTMとストックの使い分けは
LGTM→ざっと読んでみてなんとなくいいなと思ったらクリック
ストック→役に立つ情報だから保存しておいて後で何度も読み返したい記事を記録

といった形で使われてると思うのですが、こう考えると明らかにストックの方がユーザーにとって有益な記事に使われてると思います。ランキングなどでは現在LGTM数が使われてますがストック数に変更、少なくともそれを加味するようにして欲しいです。

**Comment by tanaka-asami415** [comment]
@lamrongol 
ご意見ありがとうございます🙏
ユーザーランキングは、Contribution数で計算されているため、ストックされた数も含まれているのですが
Contributionとして加算される値：LGTM>ストック
となっているため、現在はLGTMの方が重視される状態となっています。

2021年7月に、記事がストックされた場合もContributionが加算されるようになりました。
[Contribution計算に含める項目を追加します](https://blog.qiita.com/add-items-for-calculating-contribution/)

変更から約1年が経ち、様々なご意見もお聞きしています。
すぐにこのように変えます！という確約は難しいのですが、ユーザーさまにとって有益な記事がきちんと評価されるよう
今後の加算方法も慎重に設計していきたいと考えておりますので、お待ちいただけると幸いです🙇💦

**Reply by lamrongol** [reply]
@tanaka-asami415
そうだったんですか。週刊で送られてくるメールではLGTLM数順だったのでLGTM数しか考慮されてないと思いこんでました。
こちらも改善していただけたら嬉しいです。

**Reply by tanaka-asami415** [reply]
@lamrongol 
確かにメルマガはLGTM数順になっております🙇💦
メルマガの見せ方など試行錯誤していることころですので、今回頂いたご意見も合わせて検討させていただきますね💪
ありがとうございます🙏

**Reply by lamrongol** [reply]
@tanaka-asami415
おそらく一番カンタンにできる改善方法として、「ストックしたら同時にいいねもされる」ように仕様変更していただくことは可能でしょうか？
「保存のためストックはしたけど『いいね』したいとは思わない」ということはまず無いと思うので。
私事ですがなぜか自分の記事は「ストックされるけどいいねはされない」ということが多いので。

**Reply by tanaka-asami415** [reply]
@lamrongol 
ご意見ありがとうございます🙏
「ストック」は記事を保存しておくことができる機能ですが、
保存の仕方には、気になるタイトルだったので一旦ストックしておいて後で読むなど
必ずしも「ストック」＝「いいね」ではないこともあります。

以前のコメントと重複してしまいますが、有益な記事がきちんと評価されるよう、
今後の加算方法等を慎重に設計していきたいと考えておりますので、お待ちいただけると幸いです🙇

**Reply by shiracamus** [reply]
ちなみに、私のストックのカテゴリーには以下のようなのを登録してます。
- 後で読む
- 後で使う
- 後でコメントする
- リファクタリングしたい
